cannon/index.d.ts

```diff
@@ -0,0 +1,989 @@
+declare module CANNON {
```

Comment:
Missing header. also applies to other files.

just run `npm run new-package cannon`

seee https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#create-a-new-package

A header looks like:

```ts
// Type definitions for cannon <version>
// Project: https://github.com/beatgammit/base64-js
// Definitions by: YYC <https://github.com/yyc-git>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```